pkg/chartutil/load.go

```diff
@@ -214,10 +214,25 @@ func LoadFile(name string) (*chart.Chart, error) {
 	return LoadArchive(raw)
 }
 
+// LoadIgnoreFiles loads helmignore files and returns a reader.
+func LoadIgnoreFiles(ignoreFiles []string) *bytes.Reader {
+	buff := bytes.NewBuffer(nil)
+
+	for _, ignf := range ignoreFiles {
+		f, err := os.Open(ignf)
+		if err == nil {
+			defer f.Close()
+			io.Copy(buff, f)
+		}
+	}
+
+	return bytes.NewReader(buff.Bytes())
+}
+
 // LoadDir loads from a directory.
 //
 // This loads charts only from directories.
-func LoadDir(dir string) (*chart.Chart, error) {
+func LoadDir(dir string, ignoreFiles ...string) (*chart.Chart, error) {
```

Comment:
If I could I would prefer not to separate handling global and local helmignore files. It will make the logic more hidden from people looking at the code.